TunnelVisionLabs.ReferenceAssemblyAnnotator/WellKnownTypes+NullableAttributeProvidedType.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace TunnelVisionLabs.ReferenceAssemblyAnnotator
+{
+    using System.Linq;
+    using Mono.Cecil;
+
+    internal partial class WellKnownTypes
+    {
+        private sealed class NullableAttributeProvidedType : ProvidedAttributeType
+        {
+            internal NullableAttributeProvidedType()
+                : base("System.Runtime.CompilerServices", "NullableAttribute")
+            {
+            }
+
+            protected override void ImplementAttribute(ModuleDefinition module, TypeDefinition attribute, WellKnownTypes wellKnownTypes, CustomAttributeFactory attributeFactory)
+            {
+                MethodDefinition compilerGeneratedConstructor = wellKnownTypes.SystemRuntimeCompilerServicesCompilerGeneratedAttribute.Resolve().Methods.Single(method => method.IsConstructor && !method.IsStatic && method.Parameters.Count == 0);
+                attribute.CustomAttributes.Add(new CustomAttribute(wellKnownTypes.Module.ImportReference(compilerGeneratedConstructor)));
+                attribute.CustomAttributes.Add(new CustomAttribute(wellKnownTypes.MicrosoftCodeAnalysisEmbeddedAttribute.Value.Resolve().Methods.Single(method => method.IsConstructor && !method.IsStatic && method.Parameters.Count == 0)));
```

Comment:
Could add a protected helper method to add the two attributes that keep showing up.

Reply:
(And maybe the method name could help explain why they are being added)

Reply:
Updated the code to use existing `CustomAttributeFactory` methods instead.